### PR TITLE
Migrate to Platform v3 #VP-2161

### PR DESCRIPTION
**Pull request by eMazeika**
https://virtocommerce.atlassian.net/browse/VP-2161

### CODE REVIEW

File: src/VirtoCommerce.QuoteModule.Web/Controllers/Api/QuoteModuleController.cs [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+        }
+
+        /// <summary>
+        /// Get RFQ by id
+        /// </summary>
+        /// <remarks>Return a single RFQ</remarks>
+        /// <param name="id">RFQ id</param>
+        [HttpGet]
+        [Route("{id}")]
+        public async Task<ActionResult<QuoteRequest>> GetById(string id)
```
Comment (akak1977) [inline src/VirtoCommerce.QuoteModule.Web/Controllers/Api/QuoteModuleController.cs:52]: If **NotFound** result should not be interpreted as exceptional situation by client, please annonate this action method with 
`[ProducesResponseType(typeof(void), StatusCodes.Status404NotFound)]`

File: src/VirtoCommerce.QuoteModule.Web/Properties/launchSettings.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
```
Comment (akak1977) [inline src/VirtoCommerce.QuoteModule.Web/Properties/launchSettings.json:1]: Does this file really need?

Comment (eMazeika) [reply src/VirtoCommerce.QuoteModule.Web/Properties/launchSettings.json:1]: Not sure, but it's in most (or every) current v3 module

File: src/VirtoCommerce.QuoteModule.Web/Controllers/Api/QuoteModuleController.cs (hunk 1 above)
Comment (eMazeika) [reply src/VirtoCommerce.QuoteModule.Web/Controllers/Api/QuoteModuleController.cs:53]: ok

File: src/VirtoCommerce.QuoteModule.Data/Repositories/DesignTimeDbContextFactory.cs [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+namespace VirtoCommerce.QuoteModule.Data.Repositories
+{
+    public class DesignTimeDbContextFactory : IDesignTimeDbContextFactory<QuoteDbContext>
+    {
+        public QuoteDbContext CreateDbContext(string[] args)
+        {
+            var builder = new DbContextOptionsBuilder<QuoteDbContext>();
+
+            builder.UseSqlServer("Data Source=(local);Initial Catalog=VirtoCommerce;Persist Security Info=True;User ID=virto;Password=virto;MultipleActiveResultSets=True;Connect Timeout=30");
```
[the remark this answers is not in the document]
Comment (eMazeika) [reply src/VirtoCommerce.QuoteModule.Data/Repositories/DesignTimeDbContextFactory.cs]: fixed

**Pull request merged**